.github/workflows/testing-framework.yaml

```
@@ -139,8 +145,14 @@ jobs:
       matrix:
         include:
           - image: llamacpp_python
+          #- image: llamacpp_python_vulkan
+          - image: llamacpp_python_cuda
```

These are labeled wrong, `llamacpp_python` is correct though:
```suggestion
          #- image: llamacpp-python-vulkan
          - image: llamacpp-python-cuda
```